Pranay - Made Changes For Allowing Multilple Team Assigning and Deleting Still…

Pranay-ai opened the pull request:
… WIP

# Description
![image](https://github.com/user-attachments/assets/670366ff-36aa-4321-8bcd-0a061b457400)


## Related PRS (if any):
Just this previous and closed PR #[2436](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/2436)

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. log as admin user
5. Open a User Profile-> Teams->Assign Teams  
6. Now Perform Delete Teams The Functionality Should Work  without saving the changes.
7. Verify If the Changes are working for both the Desktop and Mobile Version


Video of Change


https://github.com/user-attachments/assets/f009cfb1-3520-4941-9842-e1b94945c090

JatinAgrawal94 reviewed (changes requested):
On Not using the save button the team is not removed from the list. Please refer to the video for this.

https://github.com/user-attachments/assets/e32feee8-1811-4358-82ec-cbbf585efcf1

SMitta11 reviewed (changes requested):
I assigned the team as given and 2021 Test New team is assigned and after that I deleted the team and did not save changes and team disappeared but when I refreshed the page the team was still showing, delete request was not saved.
Screeshots:
Added the team:
![Screen Shot 2024-08-23 at 11 19 33 PM](https://github.com/user-attachments/assets/5ed4ebdb-1c28-4bfc-bcd9-e7b50aeaf0cc)
Deleted the team without saving and team not showing:
![Screen Shot 2024-08-23 at 11 19 46 PM](https://github.com/user-attachments/assets/11674cf0-f9fd-4e60-9d49-c41b93d1e54d)
Refreshed the page but deleted team showing:
![Screen Shot 2024-08-23 at 11 20 03 PM](https://github.com/user-attachments/assets/e637b623-a72b-4b81-a912-f10f5d54a06a)

Pranay-ai commented:
> On Not using the save button the team is not removed from the list. Please refer to the video for this.
> 
>  PR.2607.mp4

Hey @JatinAgrawal94  I made it So that irrespective of deleting a team or Assigning a Team The changes should not be saved unless the user clicks save, this was done to make sure that accidental deletion of teams can be avoided.
Thank you

Pranay-ai commented:
> I assigned the team as given and 2021 Test New team is assigned and after that I deleted the team and did not save changes and team disappeared but when I refreshed the page the team was still showing, delete request was not saved. Screeshots: Added the team: ![Screen Shot 2024-08-23 at 11 19 33 PM](https://private-user-images.githubusercontent.com/124092470/361101576-5ed4ebdb-1c28-4bfc-bcd9-e7b50aeaf0cc.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQ1MTAxMzAsIm5iZiI6MTcyNDUwOTgzMCwicGF0aCI6Ii8xMjQwOTI0NzAvMzYxMTAxNTc2LTVlZDRlYmRiLTFjMjgtNGJmYy1iY2Q5LWU3YjUwYWVhZjBjYy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwODI0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDgyNFQxNDMwMzBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lOWY0ZmExNDhmNzFiMzk1OWUxYWVhNzc0OWI3MjIyZDgxZTdjZGRjOGQ3Yjk2NjE5MDkyZGUwYTIwNWJkMTM3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.P7Pm4KsdjuRf9C3Mrbm3S4Qp45XV14qWJy2VWgK3dE0) Deleted the team without saving and team not showing: ![Screen Shot 2024-08-23 at 11 19 46 PM](https://private-user-images.githubusercontent.com/124092470/361101596-11674cf0-f9fd-4e60-9d49-c41b93d1e54d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQ1MTAxMzAsIm5iZiI6MTcyNDUwOTgzMCwicGF0aCI6Ii8xMjQwOTI0NzAvMzYxMTAxNTk2LTExNjc0Y2YwLWY5ZmQtNGU2MC05ZDQ5LWM0MWI5M2QxZTU0ZC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwODI0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDgyNFQxNDMwMzBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1mZmFhZTRkZTk5N2M3YzkwMTNkNmJmNDBjNTE0NmI0MTU1ZTA4MjBjYjkwODFlYzFjYTYyOWM3ZGMwOGNlYmUxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.u36WpGQcxdzFuBQdy6XvE1ylBUfriFLOuEfwXgRH33s) Refreshed the page but deleted team showing: ![Screen Shot 2024-08-23 at 11 20 03 PM](https://private-user-images.githubusercontent.com/124092470/361101619-e637b623-a72b-4b81-a912-f10f5d54a06a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQ1MTAxMzAsIm5iZiI6MTcyNDUwOTgzMCwicGF0aCI6Ii8xMjQwOTI0NzAvMzYxMTAxNjE5LWU2MzdiNjIzLWE3MmItNGI4MS1hOTEyLWYxMGY1ZDU0YTA2YS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwODI0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDgyNFQxNDMwMzBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jYmViMDU1YjAzMjk3NzY3YzgzYmRhYjRiMjM2MWUzMjcxNDgwYTNhY2QxY2U0OWU4YTg1MTkyMTBiZGYxOTc2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.rhNBFgfykjaRT0ldNshtjAX9WuO3D9D24EEUFNen_Go)

Hey @SMitta11   I made it So that irrespective of deleting a team or Assigning a Team The changes should not be saved unless the user clicks save, this was done to make sure that accidental deletion of teams can be avoided.

vishnupriyaatheti reviewed (approved):
@Pranay-ai I have verified your changes and are working as expected, 
Below are the scenarios I have checked in light & dark modes with owner account.
1) Single add of team 
2) Multiple teams adding
3) Single deletion of team
4) Multiple deletion of teams
5) Combination of multiple addition and deletion of teams. All scenarios are working as expected, The changes are reflecting only after click of save button only, If I refresh the page without save the data is showing up as expected. Thankyou.  There is suggestion saving of items taking a little more time where we should work on to reduce the latency for the end user.
![FE PR 2607 after save of all added teams with one team dleted](https://github.com/user-attachments/assets/99bcc9e3-0a17-489c-8bd1-6ada0b911e6a)
![FE PR 2607 all teams added not saved](https://github.com/user-attachments/assets/8e1ecec6-a0e6-4904-92ef-ccdc8a9d6a68)

RishithaMamidala reviewed (dismissed):
@Pranay-ai 
The functionality for adding and deleting both single and multiple teams operates as expected in both normal and mobile views. Additionally, the save button works correctly as intended.

https://github.com/user-attachments/assets/176864b7-1884-4e04-9b69-89693ae8e35f

SnelPatare reviewed (dismissed):
Changes are working as expected both in Light and Dark mode. 
I tested changes by adding, deleting multiple teams. Please find below image. 

![image](https://github.com/user-attachments/assets/f2642797-76e5-4f07-aacb-75a3b52b814d)

![image](https://github.com/user-attachments/assets/9e5671fa-5fc5-4fdf-a4ef-4049fe36ba70)

oliviahyw reviewed (dismissed):
I have tested assigning teams and deleting teams, they are working as expected. However, my cursor kept flickering, not sure if it's just me. (Please check my attached screenshot)
<img width="826" alt="Screenshot 2024-08-28 at 8 31 01 PM" src="https://github.com/user-attachments/assets/7059eb7d-1ea5-4a02-9d43-ebd0cebf15a1">
<img width="849" alt="Screenshot 2024-08-28 at 8 31 11 PM" src="https://github.com/user-attachments/assets/403c6c5a-01a4-48c1-86ac-5fd1981465b0">

https://github.com/user-attachments/assets/1674ad2c-7422-41b7-b8dc-d858927a5aaf

beblicarl reviewed (commented):
The functionality worked, however, when I restarted the page the deleted buttons came back. It didn't come back after saving changes though.

[screencast-localhost_3000-2024.08.29-12_19_28.webm](https://github.com/user-attachments/assets/e7fffe56-e297-4674-8f45-a65e7d4d8c8b)

preethamdnr reviewed (approved):
Hello Pranay,

I have checked this PR and working as expected. 

https://github.com/user-attachments/assets/09728d51-d368-41aa-b4f5-47bfbf57c1a3

gowthamdongari reviewed (approved):
Hello Pranay,
I have checked this PR  and working as expected.

https://github.com/user-attachments/assets/4e1f7d56-16e9-4108-89b1-514dcc44d2eb

sri0606 reviewed (changes requested):
Hello @Pranay-ai , correct me if I'm wrong but aren't the changes (deletion/addition of teams) need to be auto-saved? Why would I need to click "Save changes" when the requirement is otherwise? I understand you are doing this to make sure users don't delete by mistake but the requirement is to make them auto-save.


https://github.com/user-attachments/assets/aef19dd8-9987-46df-8ffa-21ee56700050

vedantg24 reviewed (approved):
I tested by adding and deleting single as well as multiple teams. Changes are working as expected.

https://github.com/user-attachments/assets/1ae8240a-7242-4314-9815-ff4a48b4d324

yashwanth170 reviewed (commented):
I can add users to different teams and remove them as needed, but I have to save the changes after deleting; otherwise, the teams reappear upon reloading. Additionally, the cursor flickers while adding a user to a team, though I’m not sure if this issue is within the scope of this PR.

https://github.com/user-attachments/assets/e3872a04-ea37-49dc-ae1a-9c47f352a276

panchalvatneu commented:
> > On Not using the save button the team is not removed from the list. Please refer to the video for this.
> > PR.2607.mp4
> 
> Hey @JatinAgrawal94 I made it So that irrespective of deleting a team or Assigning a Team The changes should not be saved unless the user clicks save, this was done to make sure that accidental deletion of teams can be avoided. Thank you

I don't get it, aren't you supposed to do the opposite? Jae wants it to auto-save right? so why should I need to click save? although I do see your concern.

one-community commented:
> > > On Not using the save button the team is not removed from the list. Please refer to the video for this.
> > > PR.2607.mp4
> > 
> > 
> > Hey @JatinAgrawal94 I made it So that irrespective of deleting a team or Assigning a Team The changes should not be saved unless the user clicks save, this was done to make sure that accidental deletion of teams can be avoided. Thank you
> 
> I don't get it, aren't you supposed to do the opposite? Jae wants it to auto-save right? so why should I need to click save? although I do see your concern.

Yes, i want it to auto-save. Accidentally deleting a person from a team here is super easy to fix and constantly having to save this simple function is a waste of time.

one-community commented:
> I can add users to different teams and remove them as needed. The cursor flickers while adding a user to a team, though I’m not sure if this issue is within the scope of this PR.

Please fix this too

one-community commented:
> The functionality worked, however, when I restarted the page the deleted buttons came back. It didn't come back after saving changes though.

My requested AutoSave will fix this too

ashayk9 reviewed (dismissed):
I have tested the functionality in both light and dark modes using the admin account, and it is working as expected. The following scenarios were verified:
-Adding and deleting multiple teams
-adding and Deleting a single team
-A combination of multiple additions and deletions of teams
All scenarios performed correctly, with changes reflecting only after clicking the 'Save' button. If the page is refreshed without saving, the original data is displayed, and no additions or deletions are permanently applied unless the 'Save' button is clicked.
Although the functionality is working properly, the save is taking time. On hovering on assign teams, the cursor is flickering takes time before it converts to a pointer and I can click assign teams. I am not sure if this issue is only with me.
![Screenshot 2024-09-03 at 12 03 26 PM](https://github.com/user-attachments/assets/5ed98935-a646-4b58-ba67-3e4cc3766807)
original
![Screenshot 2024-09-03 at 12 04 25 PM](https://github.com/user-attachments/assets/7061cf41-8dd2-447c-8702-6186f31450fe)
after addition
![Screenshot 2024-09-03 at 12 04 43 PM](https://github.com/user-attachments/assets/2ea1bc36-d014-4da8-9382-d8ec9a71aca8)
after deleting
if refreshed now return to og
![Screenshot 2024-09-03 at 12 03 26 PM](https://github.com/user-attachments/assets/96609ff0-7566-4cf8-be15-62c6313d40aa)
if saved only then retains
![Screenshot 2024-09-03 at 12 06 15 PM](https://github.com/user-attachments/assets/00b9f524-fedb-408d-99f6-9d744bb976d6)

FayeLV commented:
Hi,
I have verified your changes, and they are working as expected—great job! Below are the scenarios I tested:
Single team addition
Multiple teams addition
Single team deletion
Multiple teams deletion
Combination of multiple additions and deletions of teams

All scenarios are functioning as expected. However, I have two recommendations for improvement:
Show a confirmation message when the user clicks the delete button to prevent accidental deletions.
I noticed that when I click the refresh button, all data gets deleted. It's unclear whether this is triggered by a "delete all" action or the refresh button itself.
![Snipaste_2024-09-03_21-30-30](https://github.com/user-attachments/assets/fa6fa605-dc9c-4358-a59b-974f2de34868)
![Snipaste_2024-09-03_21-32-15](https://github.com/user-attachments/assets/6ce5ba96-c30b-47b8-89fe-7364346f4c97)
![Snipaste_2024-09-03_21-32-46](https://github.com/user-attachments/assets/64dd766e-d5db-448b-b95c-b40537560d6d)
![Snipaste_2024-09-03_21-33-08](https://github.com/user-attachments/assets/4d0db528-b499-48e8-b5cf-89e9096372cf)
![Snipaste_2024-09-03_21-33-26](https://github.com/user-attachments/assets/3708d897-9ed9-4b90-94d4-633aacc9a746)
![Snipaste_2024-09-03_21-45-24](https://github.com/user-attachments/assets/514c30e1-beb5-4a27-a0f0-fa1b08d5e417)

vishavk1992 reviewed (dismissed):
I have tested this PR and i am able to assign and delete Team .

https://github.com/user-attachments/assets/82b72dd9-093f-4738-b009-36a8cb71489b

Shreyav2000 reviewed (commented):
I tested the functionality by adding one team and deleting an individual team item, and everything worked smoothly. I also tested assigning multiple teams and deleting several items at once. The functionality is working as expected in both cases. It's important to note that changes, whether assigning or deleting teams, only take effect after saving. I’ve tested this on both desktop and mobile versions, and everything looks good. I am attaching a screen recording below for the same. Thank you!

https://github.com/user-attachments/assets/fb265831-b5a6-4940-8fe3-51023a5fe3a2

swaroop-2007 commented:
@Pranay-ai 
I have tested your changes in admin user and everything is functioning as expected. I tried adding a single team and multiple teams; deleting a single team and multiple teams; and a combination of adding and deleting multiple teams. 

Everything is functioning as expected: Teams that are added or deleted persist correctly without clicking the save button, and the data remains unchanged after refreshing the page on both the laptop and mobile versions.
I have attached my screenshots below. 
![Screenshot (339)](https://github.com/user-attachments/assets/377bb21d-039e-48bc-adc4-36891d0297e4)
![Screenshot (343)](https://github.com/user-attachments/assets/f88632f6-20fd-416c-afce-20f38dcd86e0)
![Screenshot (344)](https://github.com/user-attachments/assets/5955bf4d-d4af-43ba-8398-89224e49eaae)
![Screenshot (340)](https://github.com/user-attachments/assets/e657735d-7074-403a-a453-d899cbf3ca3e)
![Screenshot (341)](https://github.com/user-attachments/assets/0cbc0d78-ed59-473a-9336-26160caca34c)
![Screenshot (342)](https://github.com/user-attachments/assets/7a08083c-f8f0-4d05-85e6-ef08d712f5a0)

sri0606 reviewed (approved):
Hey @Pranay-ai , I reviewed your new changes and auto-save works as intended. But a minor suggestion is to disable the `Save Changes` button because as of right now that button is enabled whenever changes (add/delete) are made which can be confusing.


https://github.com/user-attachments/assets/a74600f4-f9ad-4151-a165-4eb1f4acf2ea

SMitta11 reviewed (dismissed):
I tested this PR and functionality works as expected.
1. I assigned a team to user by admin account
2. On deleting the team, team name disappears and even after refreshing it is not appearing which is the expected result.
Screenshots:
![1](https://github.com/user-attachments/assets/e78cea12-a03d-4b0c-81e4-24136fe5fd0c)
![2](https://github.com/user-attachments/assets/bdf07a57-f5fe-461b-a493-0cbc843abe17)

AJAYINAVOLU reviewed (approved):
Changes are working as expected.

<img width="950" alt="PR 2607" src="https://github.com/user-attachments/assets/e702a3c6-e803-469b-ad28-d4b284d89802">
<img width="940" alt="PR 2607-1" src="https://github.com/user-attachments/assets/487a6e6b-04ff-473e-8035-9fd991b5e39b">

Thank You

Parth-tech reviewed (dismissed):
Hey Pranay,
The PR works as intended. The `Deletion` and `New Team Assignment` is auto-saved.

`Reference:`

Original State: with no teams
<img width="615" alt="Screenshot 2024-09-07 at 2 37 15 AM" src="https://github.com/user-attachments/assets/b2b02000-ed45-4c1f-8dd1-f381a8404b71">

Assignment of a team:
<img width="870" alt="Screenshot 2024-09-07 at 2 37 27 AM" src="https://github.com/user-attachments/assets/b268e7bf-d961-4463-9835-c7113663996d">


Deletion of assigned team:
<img width="615" alt="Screenshot 2024-09-07 at 2 37 15 AM" src="https://github.com/user-attachments/assets/7fe9efa1-e5b0-4fe2-b26f-40b5d83f007a">

ishangoel17 reviewed (commented):
Verified the PR and the changes are working as expected. This could be unrelated to this issue but there is no notification message/prompt on deleting the team. Below are the screenshots for reference:
![Capture2607](https://github.com/user-attachments/assets/7f09af43-2732-4451-a336-d85fb8435d0b)
![Capture2607_1](https://github.com/user-attachments/assets/b76774ad-7eb4-485c-924e-faf611c6f929)
![Capture2607_2](https://github.com/user-attachments/assets/15d55a60-925e-4eb5-80d1-bb56232e30b0)
![Capture2607_3](https://github.com/user-attachments/assets/5f5c316f-589c-4ef6-bb25-0fd7f5e76fa5)
![Capture2607_4](https://github.com/user-attachments/assets/136cc7e4-c2f3-47f4-827a-256c22ae9d99)
![Uploading Capture2607_5.JPG…]()

vedantg24 reviewed (approved):
Tested the new changes, and they are working as expected. Auto-save for addition and deletion of teams is working perfectly fine. Also, data remains the same after refreshing.

https://github.com/user-attachments/assets/1a66ef61-cf3f-49b3-aa61-6210d447c0d9

strallia reviewed (changes requested):
### These Items Have Been Tested and Works:

- Assigning and deleting teams auto-saves.
- Functionality works on desktop, tablet, and mobile view.

### Issues:

- When I refresh the page in dark mode, there is a gap on the right next to the loading skeleton template. This occurs when refreshing anywhere on the Profile page, so it's likely out of the scope of this PR. Let me know if that is the case so I can update the status of my review.

https://github.com/user-attachments/assets/739fe262-3e89-49fc-bf67-a3488a10944b

- For assigning and deleting teams, I never had to click the Save Changes button. Which makes me think the Save Changes button is unnecessary and could be removed.

saurabhshetty29 commented:
Tested the changes in both Light and Dark modes, and everything is working as expected. Verified by adding and deleting multiple teams. See the attached image for details.
<img width="1370" alt="Screenshot 2024-09-09 at 9 23 42 PM" src="https://github.com/user-attachments/assets/37c8e66d-06ad-4c81-969b-1e3b03a636fc">
<img width="1345" alt="Screenshot 2024-09-09 at 9 24 16 PM" src="https://github.com/user-attachments/assets/c55ba07f-2160-49e7-91f7-72b5d7dae1a9">
<img width="1243" alt="Screenshot 2024-09-09 at 9 25 22 PM" src="https://github.com/user-attachments/assets/2355ed14-a852-48d0-93bd-7bf814a2024f">

SFA23SCM35V commented:
I have tested assigning and deleting teams, and both functions are working as expected. However, I noticed that my cursor kept flickering.
![PR #2607](https://github.com/user-attachments/assets/3a591e9d-e1dd-4b69-bb39-df43e2c245a8)

crystallow1168 reviewed (dismissed):
Hello @Pranay-ai,

✅ Verified that I can add teams to a user  successfully and delete a user's team successfully without clicking on the "Save Changes" button. (refer to the clip below) 
✅ Works on mobile version and dark mode
Observation: There is a toast notification popup after adding a team, but there's no toast notification popup for deleting a team. 

https://github.com/user-attachments/assets/ceb956d1-0b42-4462-8523-8c9473322cfa

SFA23SCM35V reviewed (approved):
Everything works good approving
![Screenshot 2024-09-12 002119](https://github.com/user-attachments/assets/3fe895a1-8aa2-4448-b0ae-510b98dbd804)

SnelPatare reviewed (dismissed):
Changes are working as expected. Please find below screenshot. 
![image](https://github.com/user-attachments/assets/323e2709-4b9c-4803-bb2b-9282bd382a44)

nishitag5 reviewed (dismissed):
Working as expected. The addition and deletion of the teams is auto-saved. 
Tested addition and deletion a team without clicking the "save changes" button.




https://github.com/user-attachments/assets/fa76e8e2-e143-4e1f-a505-98e50af5b495

RishithaMamidala reviewed (dismissed):
@Pranay-ai 
Reviewed the changes in the PR. Can assign and delete teams without saving explicitly.
Assigning
<img width="859" alt="PR 2607" src="https://github.com/user-attachments/assets/52467252-304a-4b47-aaaf-5ba8693ae1fb">
Deleting
<img width="878" alt="PR 2607-2" src="https://github.com/user-attachments/assets/5709df80-aba4-437a-a697-ebdebd0d5b82">

sud306 commented:
Hello,
I reviewed PR2607 and was able to assign and delete teams without hitting the save button.
The changes were there even after refreshing the page.
![Uploading 10. PR2607_1.jpg…]()

![10  PR2607_2](https://github.com/user-attachments/assets/12d5952c-f39c-4119-932a-fbebb70bf1ab)
![Uploading 10. PR2607_3.jpg…]()

Muzammil603 reviewed (approved):
Its working as expected
assigning team
![image](https://github.com/user-attachments/assets/3d4674fe-dcb3-4ba3-bc0a-0a5b6535b0de)
not saving changes and deleting
![image](https://github.com/user-attachments/assets/1b06d756-37d6-4a6d-afb0-9ffbd8e68446)
refreshing the page
![image](https://github.com/user-attachments/assets/1bb6fcc9-a2dd-48eb-b928-8acb2738e7d5)

lcohen730 reviewed (changes requested):
Adding and deleting teams works fine and auto-saves, even when you create a new team, so that's good!

As others pointed out, there is a toaster notification for adding, but none for deleting. Would be good to have both for UX consistency.

Also as others pointed out, the mouse is flickering when you hover over the bottom area of the "Assign Team" button, though I realized, as you can see in my video, it seems to only flicker when there are no teams assigned. As far as I could see, it looks like there is no flickering when the user has at least one team assigned.


https://github.com/user-attachments/assets/809b42f7-e4ed-43cb-99ad-582376e123e7

mrinalini-raghavendran19 reviewed (dismissed):
Looks good to me!

The functionality works as expected - user is able to add teams after deleting unlike before where we had to commit all changes before making new changes. 

Minor suggestion: Disable autosave to avoid accidental deletion of teams.

Another issue - the cursor flickers when I delete all teams and hover over 'Assign Teams' button. 

https://github.com/user-attachments/assets/003009ec-9508-416a-8d7f-0d46ee45df1c

javid679 commented:
The functionality for adding and deleting both single and multiple teams operates as expected in both normal and mobile views. Additionally, the save button works correctly as intended.


https://github.com/user-attachments/assets/35820540-42b2-44d1-9441-a6c4c4221368

pallavithorat commented:
The features for adding and removing both individual and multiple teams are functioning properly, and the save button works as expected.
<img width="1470" alt="PR #2607" src="https://github.com/user-attachments/assets/cdda85d6-1317-4f8a-8690-13f6dcd74025">

yany960 commented:
This PR is outdated and has been replaced by PR #2806 , which has already been merged. Closing this PR to keep the repository clean.
